pkg/arvo/app/publish.hoon

```diff
@@ -1741,33 +1740,53 @@
               title.u.book
               description.u.book
           ==
+          %-  metadata-poke
```

Comment:
You probably want backstepped `::` around your long-form `:~` entries at this point. Same applies below.